images/access-setup/content/bin/setup_compute.sh

```diff
@@ -136,6 +138,38 @@ check_prerequisites() {
   fi
 }
 
+generate_shared_manifests(){
+  printf "[Shared manifests]\n"
+
+  printf "    - tekton-chains signing key: "
+  manifest="$manifests_dir/compute/tekton-chains/signing-secrets.yaml"
+  if [ ! -e "$manifest" ]; then
+    printf "\n"
+    manifests_tmp_dir="$(dirname "$manifest")/tmp"
+    mkdir -p "$manifests_tmp_dir"
+    cosign_passwd="$( head -c 12 /dev/urandom | base64 )"
+    echo -n "$cosign_passwd" > "$manifests_tmp_dir/cosign.password"
+    cosign_image="quay.io/redhat-appstudio/appstudio-utils:eb94f28fe2d7c182f15e659d0fdb66f87b0b3b6b"
+    podman run \
```

Comment:
CI is failing because `podman` is not installed:
```
setup_compute.sh: line 153: podman: command not found
```

Reply:
Thanks, @xinredhat is having a look at the issue.